docs/migration/3.0.0-beta-to-rc.mdx

```diff
@@ -0,0 +1,250 @@
+# Migration Guide
+
+Widgetbook `3.0.0-rc.1` is out :tada:! This release candidate marks one step closer towards the stable version of Widgetbook 3. It's like getting the burgers out of the freezer but not putting them on the grill yet. A lot of breaking changes were made compared to the beta version of Widgetbook 3. These changes were necessary to improve the DX and the integration with Widgetbook Cloud.
+
+## Highlights
+
+1. No more code generation for addons or app builders, only for use cases.
+2. Almost zero-dependencies, after removing dependency on `provider`, `go_router` and `flutter_bloc`.
+3. Better addons and knobs deep linking on the web.
+4. New addons API that make your `appBuilder` much cleaner.
+5. Performance optimization by minimizing rebuilds.
+6. Some bug fixes, yet a lot more to come.
+
+## How to Migrate
+
+### Annotations
+
+This release candidate comes with code generation support for use cases only. So the following changes were made:
+
+1. Removed Annotations
+
+   | Annotation                         | Alternative                           |
+   | :--------------------------------- | :------------------------------------ |
+   | `@WidgetbookLocales`               | `LocalizationAddon`                   |
+   | `@WidgetbookLocalizationDelegates` | `LocalizationAddon`                   |
+   | `@WidgetbookTheme`                 | `ThemeAddon`                          |
+   | `@WidgetbookAppBuilder`            | `Widgetbook`'s `appBuilder` parameter |
+
+1. `@WidgetbookUseCase` has been renamed to `@UseCase`.
+
+1. `@WidgetbookApp` has been replaced with `@App` that takes no parameters. The new `@App` annotation generates a file containing a single variable called `directories`.
+
+### `Widgetbook` Configuration
+
+Since the `Widgetbook` app is no longer generated by `@App`, you will need to configure your app as follows:
+
```

Comment:
Good, but maybe add another sentence pointing out that the `widgetbook.g.dart` file will be generated and that you get the `directories` variable from that file.